### PR TITLE
Modal

### PR DESCRIPTION
**Pull request by Megha-Joshi**
Screenshot (93)
Screenshot (94)

This is the Modal component of my CL. Please review it!

### CODE REVIEW

File: Components/Modal/modal.css [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    align-items: center;
+    border-radius: var(--container-border-radius);
+    margin: var(--container-margin);
+    padding: var(--container-padding);
+}
+
+.modal-section {
+    display: flex;
+    flex-direction: column;
+    gap: 0.25rem;
```
Comment (Muzammil-khan710) [inline Components/Modal/modal.css]: Use px for small values

**Review by Muzammil-khan710 — APPROVED** [review]
Awesome UI 
Code is well structured also good namings of classes have been used
Good to merge!

File: Components/Modal/modal.css [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    border-radius: var(--container-border-radius);
+    font-weight: 400;
+    padding: var(--container-padding);
+    box-shadow: 2px 2px 6px 0px rgba(0, 0, 0, 0.1);
+
+}
+
+.modal-content {
+    padding: 0.75rem 0.5rem 0.5rem 0.5rem;
+}
```
Comment (Shubhamkr2610) [inline Components/Modal/modal.css:32]: use px for less than 1 rem

File: Components/Modal/modal.html [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+                <li>
+                    <a href="https://twitter.com/megha24joshi"><i class="fab fa-twitter"></i></a>
+                </li>
+            </ul>
+        </nav>
+    </div>
+
+
+    <!-- Docs -->
+    <div class="main">
```
Comment (Shubhamkr2610) [inline Components/Modal/modal.html:43]: Appreciation: it good to see that you are giving comment message , it helps to reviewer while reviewing code

**Review by iRohitGaur — APPROVED** [review]
Nice work! Good to merge 👍🏻

**Review by Shubhamkr2610 — COMMENTED** [review]
code is clean and readable . Easy to understand . you can merge it

**Review by Shubhamkr2610 — APPROVED** [review]

File: Components/Modal/modal.css (hunk 2 above)
Comment (zoyatanweer) [reply Components/Modal/modal.css:32]: use px here

**Review by zoyatanweer — APPROVED** [review]
- very clean code 
- nice use of comments
- great UI
- good to merge

File: Components/Modal/modal.css (hunk 1 above)
Comment (Megha-Joshi) [reply Components/Modal/modal.css]: Ok

File: Components/Modal/modal.css (hunk 2 above)
Comment (Megha-Joshi) [reply Components/Modal/modal.css:32]: Done

**Pull request merged**